camayoc/tests/qcs/test_profiles.py

```diff
@@ -113,31 +168,71 @@ def test_negative_create_missing_data(cleanup):
             b) hosts
             c) credential id's
     :expectedresults: Error is thrown and no new profile is created.
-    :caseautomation: notautomated
+    :caseautomation: automated
     """
-    pass
+    cred = HostCredential(client=shared_client, password=uuid4())
+    cred.create()
+    cleanup.append(cred)
+    profile = NetworkProfile(
+        client=api.Client(response_handler=api.echo_handler),
+        hosts=['localhost'],
+        credential_ids=[cred._id],
+    )
 
+    # remove hosts from fields
+    delattr(profile, data)
+    create_response = profile.create()
+    assert create_response.status_code in range(400, 500)
```

Comment:
I think you can assert for the specific response code. An specific response code on a REST API says a lot about the result and we should ensure that. If more than one response code is returned then maybe you can select the appropriate one depending on the deleted field.